.pre-commit-config.yaml

```diff
@@ -1,97 +1,103 @@
 exclude: ^misc/bench.py$
 repos:
```

Comment:
I guess we should add yamllint to pre-commit config

This should be fairly simple given yamllint repo contains a [pre-commit config](https://github.com/adrienverge/yamllint/blob/master/.pre-commit-hooks.yaml)
```yaml
- repo: https://github.com/adrienverge/yamllint
  rev: v1.26.3
  hooks:
  - id: yamllint
```

Reply:
I've planned that in another PR